### ISSUE TITLE
[Discussion] Header/footer: color contrast ratio is too low

### ISSUE THREAD

**Issue by zichy** [corona-warn-app/cwa-website#16]
## Where to find the issue
The issue can be found in the header and footer on every site.

## Describe the issue
The contrast ratio of foreground/background color is too low (1.77:1). This makes it hard for some people with visual impairment to read the text. According to [WCAG 2.1](https://www.w3.org/TR/WCAG21/#contrast-minimum), it should be at least 4.5:1 for normal text and 3:1 for large text.

See: https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=80CDEC

The actual contrast ratio is hard to determine since there's a color gradient. The ratio of white text on red background would be fine (6.27:1), but almost all of the content area is in brighter colors.

## Suggested change
Either background gradient or text should be in darker colors. Maybe it would be possible to use text shadow or a solid background color for some parts of the header/footer, e.g. main navigation.

**Comment by MalteJ** [comment]
Is this just about the Corona-Warn-App logo? Every other asset has a higher contrast as far as I can see.

**Comment by zichy** [comment]
It's about the other text as well. As I said before, the ratio is hard to determine since it's a color gradient. The ratios differ from 1.77:1 to 6:27:1 and depend on the viewport as well.

**Comment by mojoaxel** [comment]
I think the is the most problematic part:
![image](https://user-images.githubusercontent.com/600565/86393265-136cb480-bc9d-11ea-8a6a-98ca38534027.png)

**Comment by svengabr** [comment]
The colors for linear gradients were adjusted with the latest PR https://github.com/corona-warn-app/cwa-website/pull/2660 

The team is currently actively working on resolving all found WCAG issues.
